docs/developinstallwin.rst

```diff
@@ -0,0 +1,147 @@
+up, you are good to go.
+
+If wanted: Integrate conda environment into editor of your choice (i.e. PyCharm, Spyder or any other)
+
+Head over to :ref:`gettingstarted:First run` for the next steps.
+
+Using QGIS from Conda
+^^^^^^^^^^^^^^
+It might be wanted to both have a script installation and the plugin installed on your machine at the same time.
+Unfortunately, installing AvaFrame on Windows via QGIS plugin might overrule your script AvaFrame installation, causing
+issues when trying to work on both versions in parallel. This depends on your mode of installation (direct installer, conda, OSGeo4W...).
+
+Below, we have compiled a **short guide**, describing a way to install QGIS and the Connector plugin in a separate
+*conda* environment. This method will result in two script installations (if the steps above have been followed),
+where one will be used by QGIS and the other may be worked on separately, preventing any overlap.
+
+Firstly, make sure that if you have another QGIS installation somewhere, that the AvaFrame plugin is not installed.
+Next, create and activate a new environment, which will contain your QGIS installation. The name of the environment
+can be anything, but since we will install the latest version of QGIS, we will call it *qgis_latest* in this example.
+Open your conda terminal and use::
+
+  conda create -n qgis_latest
+  conda activate qgis_latest
+
+To **install the latest QGIS version**, use::
+
+  conda install qgis --channel conda-forge
+
+.. Note::
+  Because conda will always try to install the latest version, if you want to use another version, you need to specify it, e.g.:
+
+  ``conda install qgis=3.34.11 --channel conda-forge``
+
+Now install avaframe into the qgis environment. This is to make sure that the plugin will find this version first, and install the necessary dependencies.
+In the conda terminal, use::
+
+  pip install avaframe
+
+Now find the directory where your environment is located. The path should be something like:
+``C:\Users\USER\miniconda3\envs\qgis_latest``. Change into it::
+
+  cd \miniconda3\envs\qgis_latest
+
+.. Note::
+  **Tip:** To find the path, you can do ``pip show avaframe`` and copy the
+  first part which shows your environment directory
+
+Clone the AvaFrame repository and change into it::
+
+  git clone https://github.com/avaframe/AvaFrame.git
+  cd AvaFrame
+
+.. Note::
+  This will pull the latest developer version of AvaFrame, if you want the current release version for the plugin,
+  you need to specify the version, e.g.:
+
+  ``https://github.com/avaframe/AvaFrame/tree/1.8.3``
+
+Compile the cython com1DFA part and create an editable avaframe version in your environment::
+
+  python setup.py build_ext --inplace
+  pip install -e .
+
+To **run QGIS**, in the terminal, activate the environment (if not activated already) and run the ``qgis`` command::
+
+  conda activate qgis_latest
+  qgis
+
```

Comment:
Mh, I would only write 'Run qgis: ' 

If people followed the guide, the environment should be activated. If not, they installed AvaFrame into a systemwide directory, leading to problems...

Reply:
Sure, for the installation it's not needed to activate the env again. I was thinking for people just to refer back on how to open qgis again, after pc restart etc., to make sure again to activate the env. Could alternatively move the info downwards..

Reply:
In my experience it is better to keep the explanations as simple as possible: having the activate a second time here could lead to a 'double stacked' environment, which should not be a problem, but could be.... In the context/flow of this explanation please leave the activate out here. 
And furthermore: I'm hoping that anyone who is attempting this has at least the presence to get familiar with conda in case they don't know it already. So they should be aware of the fact that an environment has to be activated before they can use it again. Especially since QGis won't be found without the env...